orga/tests/integration/components/campaign/create-form_test.js

```diff
@@ -353,6 +354,173 @@ module('Integration | Component | Campaign::CreateForm', function (hooks) {
         assert.contains(t('common.target-profile-details.results.common'));
       });
     });
+
+    module('chevron', function () {
+      module('when the user click in the target profile input ', function () {
```

Comment:
**thought :** Je sais pas si ça vaut le coup d'avoir ce test.
On vérifie juste que il y a un élément avec le bon css
pareil pour les tests 'should display chevron up', 'should display the button to clear the input' et tous les tests o* on vérifie qu'un élément ou il y a bien une classe css